style.css

```diff
@@ -57,9 +58,6 @@ body {
   width: 11px;
   height: 10px;
   position: absolute;
-
-  /* top: 10%;
-  left: 30%; */
   top: 100px;
   left: 132.98px;
 }
```

Comment:
Kindly, I suggest you use more descriptive class names so that in the future you or a partner can identify them more easily.